index.html

```diff
@@ -0,0 +1,247 @@
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <!--<section id="portfolio" class="project-section">-->
+            <div>
+                <section class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project1.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h2 class="title">Tonic</h2>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
+                            <p>
+                                A daily selection of privately personalized
+                                reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <ul class="skills">
+                            <li class="skill set1">html</li>
+                            <li class="skill set2">css</li>
+                            <li class="skill set3">javascript</li>
+                        </ul>
+                        <div class="button-click">
+                            <button type="button" class="btn">See Project</button>
+                        </div>
+                    </div>
+                </section>
+
+                <div class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project2.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Multi-Post Stories</h1>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
+                            <p>
+                                A daily selection of privately personalized
+                                reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <ul class="skills">
+                            <li class="skill set1">html</li>
+                            <li class="skill set2">css</li>
+                            <li class="skill set3">javascript</li>
+                        </ul>
```

Comment:
- [ ] This technology list is not well aligned to the left. See images below 👇🏾 
- [ ] Also, notice how the text (HTML, CSS, etc) inside their container is centered in the Figma design but in your work they are not.
- [ ] Also, notice the space between each element. Kindly compare your code and ensure it matches the figma design as much as possible. 
Figma design | Your work
---- | ----
![image](https://user-images.githubusercontent.com/84156714/228533886-286754ff-6373-4b9d-87b1-7ff48d25f3de.png) | ![image](https://user-images.githubusercontent.com/84156714/228534028-f83ab530-b172-4e89-94e8-08138fd336a5.png)